suite-common/wallet-utils/src/validationUtils.ts

```diff
@@ -62,6 +62,9 @@ export const isTaprootAddress = (address: string, symbol: Account['symbol']) =>
     );
 };
 
+export const isCashAddressUppercase = (address: string) =>
+    /^(bitcoincash|bchtest|bchreg):/.test(address.toLowerCase()) && /[A-Z]/.test(address);
```

Comment:
No need to check for `bchtest` and `bchreg` as they are not supported in Suite and never will be. Please remove also `tltc1` and `tvtc1` from `isBech32AddressUppercase` and shorten unit tests accordingly.